### DISCUSSION TITLE
How to update Meshcentral

### DISCUSSION THREAD

**Discussion by Nomex1337** [Ylianst/MeshCentral#6245] [General]
Hi,

i have Meshcentral running on Synology Nas /container manager
wenn i login to meshcentral an do the update nothing is happened . What i need to do?

![image](https://github.com/Ylianst/MeshCentral/assets/156704707/4553a60d-bb48-49c5-8eac-77a81c47af7d)

**Comment by si458** [comment]
im assuming the nas is using docker, so you update the docker way and dont use the web ui!
so update your docker-compose image from 1.1.16 to 1.1.24
you might need to share what the web ui looks for your container settings from the NAS

**Comment by Nomex1337** [comment]
![image](https://github.com/Ylianst/MeshCentral/assets/156704707/d00d0f1f-ce40-439d-b67b-535c831cebe4)

![image](https://github.com/Ylianst/MeshCentral/assets/156704707/87c27ddb-d5f1-47dd-b4d3-a53728f2302d)

![image](https://github.com/Ylianst/MeshCentral/assets/156704707/1bfa428f-25ee-40b9-8cc3-a7348a053fc9)
